skbio/stats/ordination/tests/data/vare_rda_constraints_from_vegan.csv

```diff
@@ -0,0 +1,25 @@
+"RDA1","RDA2","RDA3","RDA4","RDA5","RDA6"
```

Comment:
I'd try to remove these files from this PR and keep them in #1529 .  It'll make it easier for reviewing.